### DISCUSSION TITLE
Print formatted numbers

### DISCUSSION THREAD

**Discussion by ghost** [objeck/objeck-lang#200] [Q&A] [answered]
> There are ways to adjust the `Float` precision and format (i.e., fixed, scientific, and hex).

How to do that?

**Comment by objeck** [comment, marked as answer]
Look at the Standard class.

**Reply by ghost** [reply]
`Number->Format`?

**Reply by ghost** [reply]
Oh I see it, `System.IO.Standard`

**Comment by objeck** [comment]
I need to clean up the auto docs for ``ConsoleIO``references as it is a system class; here's an example.

``Standard->SetFloatPrecision(2)->Print(3.145)->(" pie")->PrintLine();``
